workflows/speciesabundance.nf

```diff
@@ -68,9 +69,17 @@ workflow SpAnce {
                 fastq_2 ? tuple(meta, [ file(fastq_1), file(fastq_2) ]) :
                 tuple(meta, [ file(fastq_1) ])}
 
+    ch_kraken2_db = Channel.fromPath( "${params.kraken2_db}", type: 'dir')
```

Comment:
I also think this should be `Channel.value(file(("${params.kraken2_db}"))` instead of `Channel.fromPath`, since otherwise the database would only be passed to the first of a collection of samples.

Similar to how SNVPhyl sets the reference genome: https://github.com/phac-nml/snvphylnfc/blob/main/workflows/snvphylnfc.nf#L224

Reply:
Changed in [d39bca1](https://github.com/phac-nml/speciesabundance/pull/5/commits/d39bca1119d3763be3511f4b21008368189a3b12)
Thank you for catching this one!